api/courses/1/dashboard.json

```diff
@@ -83,6 +83,63 @@
             "complete_exercise_count": 7
         }
     ],
+    "plans": [
+        {
+            "id": "1",
+            "type": "reading",
+            "title": "Read Pages 9-11",
+            "opens_at": "2015-03-04T16:40:23.796Z",
+            "due_at": "2015-03-10T04:00:00.000Z",
+            "settings": {
+                    "page_ids": ["3", "2", "1"]
+            }
+        },{
+            "id": "3",
+            "type": "reading",
+            "title": "Read Other Things",
+            "opens_at": "2015-02-28T16:40:23.796Z",
+            "due_at": "2015-03-01T04:00:00.000Z",
+            "settings": {
+                    "page_ids": ["3", "2", "1"]
+            }
+        },{
+            "id": "2",
+            "type": "homework",
+            "title": "Complete exercise 12A on pg 12",
+            "opens_at": "2015-03-04T11:40:23.796Z",
+            "due_at": "2015-03-18T04:00:00.000Z",
+            "settings": {
+                    "exercise_ids": ["1", "3", "5", "7", "9", "11", "13", "15", "17", "19", "21", "23", "25"],
+                    "exercises_count_dynamic": 2
+            }
+        },{
+            "id": "5",
+            "type": "homework",
+            "title": "Complete exercise 12A on pg 12",
+            "opens_at": "2015-03-31T11:40:23.796Z",
+            "due_at": "2015-04-01T04:00:00.000Z",
+            "settings": {
+                    "exercise_id": ["11"]
```

Comment:
good point, also, looks like settings isn't supposed to be in this endpoint anyway